package.json

```diff
@@ -16,6 +16,7 @@
     "build:react-components": "yarn workspace @guardian/source-react-components build",
     "build:storybook": "build-storybook -o dist",
     "build:storybook:ci": "NODE_ENV=production build-storybook -o dist",
+    "deploy:storybook": "storybook-to-ghpages",
```

Comment:
is this something we want in the main scripts? does this mean i could run it from my machine too? somehow this feels like something you should only be able to do from github?

Reply:
It's hard to make sure that this can only be run from GitHub, but if we keep it out of scripts, it makes it less obvious 😄 👍